### PR TITLE
Add Date Range Field In Aggregation Filters

### PR DESCRIPTION
**Pull request by ZChristine**

### CODE REVIEW

File: concepts/search-concept-aggregation.md [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
 Refine search results and show their distribution in the index.
 
 ## Example 1: Request aggregations by string fields
 
-The following example searches **listItem** resources and aggregates results by their file type and content class, both of which are string values.
+The following example searches **listItem** resources and aggregates results by their file type, content class, and last modified time, all of which are string values.
 
 The response includes two [searchBucket](/graph/api/resources/searchbucket?view=graph-rest-beta&preserve-view=true) objects for the two aggregations:
-- The **key** property specifies the actual value (by `FileType` or `contentclass`) for those matching **listItem** objects that are aggregated in the same bucket by that value.
+- The **key** property specifies the actual value (by `fileType`, `contentclass`, or `lastModifiedTime`) for those matching **listItem** objects that are aggregated in the same bucket by that value.
```
Comment (Lauragra) [inline concepts/search-concept-aggregation.md:18]: For consistency I changed FileType to fileType, although it's FileType in the example. Please verify which is should be; the pattern is generally lower camel case.

File: concepts/search-concept-aggregation.md [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
-title: "Use the Microsoft Search API in Microsoft Graph refine queries with aggregations"
+title: "Use the Microsoft Search API in Microsoft Graph to refine queries with aggregations"
 description: "You can use the Microsoft Search API to retrieve aggreations"
 author: "nmoreau"
 ms.localizationpriority: medium
 ms.prod: "search"
 ---
 
-# Refine search results using aggregations
+# Use the Microsoft Search API in Microsoft Graph to refine queries with aggregations
```
Comment (Lauragra) [inline concepts/search-concept-aggregation.md:10]: This article can use a richer introduction to help set the context before providing the examples. Can you give some more details about the overall scenario here?

File: concepts/search-concept-aggregation.md (hunk 1 above)
Comment (ZChristine) [reply concepts/search-concept-aggregation.md:18]: You're right, let's use the lower camel case for consistency as other aggregation field. In the actual scenario, the case will change corresponding as we set it.

File: concepts/search-concept-aggregation.md (hunk 2 above)
Comment (ZChristine) [reply concepts/search-concept-aggregation.md:10]: added

**Review by Lauragra — APPROVED** [review]

**Pull request merged**